oncn-mda/cli_src/func/global.c

```diff
@@ -158,11 +182,19 @@ static int get_map_fd(const char *const map_name, int *const fd)
     return SUCCESS;
 }
 
+#if LIBBPF_HIGHER_0_6_0_VERSION
+static int init_mesh_map(
+    struct mesh_map_info *const fds_map,
+    const char *const pin_file_path,
+    const char *const map_name,
+    struct create_map_attr *const map_attr)
+#else
```

Comment:
just include create_map_attr

Reply:
> just include create_map_attr

Yes, this can make code logic clearer，DONE

Reply:
> just include create_map_attr

Yes, this can make code logic clearer，DONE